sdk/metrics/build.gradle.kts

```diff
@@ -48,3 +46,7 @@ tasks {
     dependsOn(testing.suites)
   }
 }
+
+animalsniffer {
+  ignore("java.util.concurrent.atomic.DoubleAdder", "java.util.concurrent.atomic.LongAdder")
```

Comment:
I tried using the `@IgnoreJRERequirement` annotation on the classes that use `j.u.c.a.DoubleAdder` and `j.u.c.a.LongAdder` but it didn't work. It got rid of of some of the animal sniffer errors, but there was always one pesky error remaining.

Added the annotation at the class level like so:
```
@IgnoreJRERequirement
final class JreLongAdder implements LongAdder {

  private final java.util.concurrent.atomic.LongAdder delegate =
      new java.util.concurrent.atomic.LongAdder();
...
```

And still see errors like:
```
[Undefined reference] io.opentelemetry.sdk.metrics.internal.concurrent.(JreDoubleAdder.java::1)
  >> java.util.concurrent.atomic.DoubleAdder
```

Not sure what I'm doing wrong, but this allowed me to get around the errors with what I believe is a safe implementation.

Reply:
I will play around a bit too. A Gradle based solution is OK too but I hope we can specify our classes to ignore from checking, not the JDK classes themselves. Otherwise the latter can easily get used outside those classes accidentally.

Reply:
> Otherwise the latter can easily get used outside those classes accidentally.

Yeah exactly. Definitely prefer to do this via annotation if possible.